docs/book/src/developer/testing.md

```diff
@@ -267,7 +267,6 @@ kustomize_substitutions:
   EXP_CLUSTER_RESOURCE_SET: "true"
   EXP_KUBEADM_BOOTSTRAP_FORMAT_IGNITION: "true"
   EXP_RUNTIME_SDK: "true"
-  EXP_LAZY_RESTMAPPER: "true"
```

Comment:
✨✨✨✨✨

I'll stop counting now, I love feature gate removals.

Reply:
Yeah we should do it more often :)